tools/release-plugin.sh

```diff
@@ -119,6 +119,21 @@ fi
 
 proceed_p "Releasing $PROJECT $VERSION" "Proceed?"
 
+# Check if a remote branch for the release branch exits and ask to delete it if it does.
+PREFIX=$(jq -r '.extra["release-branch-prefix"] // empty' "$BASE"/projects/"$PROJECT"/composer.json)
```

Comment:
It makes zero difference, but since we expect the default to be empty-string I'd do that instead of using `empty` here.
```suggestion
PREFIX=$(jq -r '.extra["release-branch-prefix"] // ""' "$BASE"/projects/"$PROJECT"/composer.json)
```